### DISCUSSION TITLE
Enhance the web responsive images support

### DISCUSSION THREAD

**Discussion by JeremieLitzler** [vuepress-theme-hope/vuepress-theme-hope#2851] [Q&A] [answered]
Hi @Mister-Hope 

I was wondering if you are thinking about having more responsive images using `src-set`.

I came across [this article](https://dev.to/ascorbic/a-minimal-multi-framework-responsive-image-component-3iop) on the Netlify blog that looks interesting.

Thanks.

**Comment by Mister-Hope** [comment, marked as answer]
VuePress support setting src-set on `<img>` tag, you can just do that if you want to.

Edited: from your article, the first thing you need is to have a CDN supports chopping images with params.

> A great way to solve this is with an image CDN, which resizes the image on the fly.

You could build a component for this usage yourself, if you do have a image CDN supporting this.
